src/usr/local/www/system_advanced_network.php

```diff
@@ -274,6 +281,15 @@
 ))->setHelp('This option will suppress ARP log messages when multiple interfaces '.
 	'reside on the same broadcast domain.');
 
+$section->addInput(new Form_Checkbox(
+	'ip_change_kill_states',
+	'Reset all states',
+	'Reset all states if WAN IP changes',
+	isset($pconfig['ip_change_kill_states'])
+))->setHelp('This option will reset all states if a WAN IP changes instead of only '.
+    'reset states associated to old IP. This can help to kill zombie states associated '.
```

Comment:
Remove "Reset" from this line.
"to old IP" should be "with the previous IP address"
Remove the last sentence (end of this line and the entire next line)